test/checks/design/InterfaceIsTypeCheckTest.hx

```diff
@@ -0,0 +1,40 @@
+package checks.design;
+
+import checkstyle.checks.design.InterfaceIsTypeCheck;
+
+class InterfaceIsTypeCheckTest extends CheckTestCase<InterfaceIsTypeCheckTests> {
+
+	public function testWithJustVars() {
+		assertMsg(new InterfaceIsTypeCheck(), InterfaceIsTypeCheckTests.TEST1, "Interfaces should describe a type and hence have methods");
```

Comment:
Personally I like the nice colour highlight and code completion but doesn't really matter.

<img width="525" alt="screen shot 2016-03-08 at 14 41 11" src="https://cloud.githubusercontent.com/assets/4947424/13604709/f2856402-e53b-11e5-8c08-fc412aeab7d9.png">